lib/auth.ts

```diff
@@ -61,7 +61,26 @@ export async function getSession(
             refreshToken = storedRefreshToken;
         } else if (refreshToken) {
             setSessionStorage(refreshToken, projectId);
-            window.history.replaceState({}, window.document.title, window.location.pathname);
+
+            // Removes access_token and refresh_token from the URL
+            // Required otherwise other useful link params are lost
+            const url = new URL(window.location.href);
+            if (url.hash.includes("access_token") || url.hash.includes("refresh_token")) {
+                const newHash = url.hash
+                    .substring(1)
+                    .split("&")
+                    .filter(
+                        (param) =>
+                            !param.startsWith("access_token") && !param.startsWith("refresh_token"),
```

Comment:
Just need to add the `=` at the beginning to avoid keys like `access_token_banana` from being filtered and LGTM
```suggestion
                            !param.startsWith("access_token=") && !param.startsWith("refresh_token="),
```